<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Empleado</class>
 <widget class="QWidget" name="Empleado">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Empleado</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>recursos/trespatitos.jpg</normaloff>recursos/trespatitos.jpg</iconset>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>391</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lblCedula">
      <property name="text">
       <string>Cedula:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblNombre">
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblApellidos">
      <property name="text">
       <string>Apellidos:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lblTelefono">
      <property name="text">
       <string>Telefono:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lblDireccion">
      <property name="text">
       <string>Direccion:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lblPuesto">
      <property name="text">
       <string>Puesto:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="lblFechaIngreso">
      <property name="text">
       <string>Fecha de Ingreso:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtCedula"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtNombre"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txtApellidos"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="txtTelefono"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="txtDireccion"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="txtPuesto"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="txtFechaIngreso"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lblTitulo">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Empleado</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>420</y>
     <width>471</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnGuardar">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>recursos/save.jpeg</normaloff>recursos/save.jpeg</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnModificar">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>recursos/editar.jpg</normaloff>recursos/editar.jpg</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEliminar">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>recursos/delete.webp</normaloff>recursos/delete.webp</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancelar">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>recursos/salir.jpg</normaloff>recursos/salir.jpg</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lblImagen">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>100</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>recursos/avatar.jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>961</width>
     <height>192</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
